<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BankSimul</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_saldo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Näytä Saldo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tapahtumat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Selaa Tilitapahtumia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ulos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Kirjaudu Ulos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nosta">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nosta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_euro">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>€</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_talleta">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Talleta</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_talleta">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nosta">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
